espressif/vscode-esp-idf-extension
# add idfTarget to project-conf editor

## Description

Add IDF Target to Project Configuration editor and object.

Move project configuration logic to project-specific configuration file.

## Type of change

- New feature (non-breaking change which adds functionality)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "ESP-IDF: Open Project Configuration"
2. Add a new profile using IDF Target and monitor baud rate
3. Try using extension command after selecting the new profile and see if values are being used.

- Expected behaviour:

- Expected output:

## How has this been tested?


**Test Configuration**:
* ESP-IDF Version: 5.0
* OS (Windows,Linux and macOS): macOS

## Checklist
